scripts/update/Updater.php

```diff
@@ -87,6 +87,6 @@ public function update($initialVersion)
             $this->setVersion('0.2.0');
         }
 
-        $this->skip('0.2.0', '2.2.1');
+        $this->skip('0.2.0', '3.0.0');
```

Comment:
This one or the version in manifest.php is not correct.

Reply:
my bad